mne/gui/_coreg_gui.py

```diff
@@ -1712,9 +1714,6 @@ def _fid_panel_default(self):
     def _headview_default(self):
         return HeadViewController(system='RAS', scene=self.scene)
 
-    def _view_options_fired(self, info):
-        self.view_options_panel.edit_traits()
-
```

Comment:
It looks like this caused the window to be opened twice (for me), because it is also opened by `CoregPanelHandler`

Reply:
Oh yes, I should have removed this, good catch! (BTW I moved this to the handler so that I could properly parent the window. That way closing the main window closed the children.)

Reply:
Ah nice! I was also wondering whether we might be able to make command-w close the window...